capstone-frontend/src/components/ListPage.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+
+class ListPage extends Component {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      gBooks: [],
+      loading: true,
+      empty: false
+    }
+  }
+
+
+  fetchBooks = async () => {
+
+    const userID = window.localStorage.getItem("userID");
+
+    console.log("Fetch Books LIST PAGE")
+
+    const bookLists = await fetch(`/api/booklist?userID=${userID}`, {
+      method: "GET",
+    }).then(resp => resp.json());
+
+    console.log("Booklist", bookLists)
+
+    const bookList = Array.from(bookLists).filter(bookList => (bookList.id === this.props.match.params.id))[0].gbookIDs;
+
+    console.log("Booklist", bookList)
+
+    if (bookList.length === 0) {
+      this.setState({loading: false, empty: true});
```

Comment:
Do we need the `empty` field in the state? Couldn't you just check the length of the `gBooks` array you create?

Reply:
Ok

Reply:
Actually, I am getting weird issues when I remove the empty state.